### ISSUE TITLE
Product page tabs issue - Magento 2.3.3

### ISSUE THREAD

**Issue by Userr017** [magento/magento2#25053]
### Preconditions (*)

1. Magento 2.3.3
2. PHP 7.2, 7.3

### Steps to reproduce (*)

1. Click open someone tab on the product page.
example. description, more information etc...
![tab](https://user-images.githubusercontent.com/26794034/66779789-c612a200-eed7-11e9-8350-1788622d81bb.png)

### Expected result (*)

1. Click open someone tab on the product page
2. Page doesn't automaticully scroll down

### Actual result (*)

1. Click open someone tab on the product page
2. Page automaticully scroll down

**Comment by Userr017** [comment]
Hope so there is a temporary solution to fix this issue soon?

**Comment by krishprakash** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @krishprakash. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @krishprakash, here is your [Magento instance](https://i-25053-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-25053-2-3-develop.instances.magento-community.engineering/admin](https://i-25053-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @krishprakash 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-21830` were created

**Issue Available**: @krishprakash, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by mackieee** [comment]
Am I correct in thinking this is affecting any is component that uses the mage collapsible/tabs library? As it is seems this is not limited to the product detail tabs.

**Comment by tantranvn** [comment]
Hi, I was having the same issue with collapsible tabs after upgraded to Magento version 2.3.3. 

My temporary solution was changing _scrollToTopIfVisible function in this file
\lib\web\mage\collapsible.js

From
```
        _scrollToTopIfVisible: function (elem) {
            if (!this._isElementOutOfViewport(elem)) {
                elem.scrollIntoView();
            }
        },
```
to

```
        _scrollToTopIfVisible: function (elem) {
            if (this._isElementOutOfViewport(elem)) {
                elem.scrollIntoView();
            }
        },
```

Hope this helps.

**Comment by boban-dj** [comment]
mm i tried but did not fix the navigation with demo data installed
see here example: https://ibb.co/pXWj8Yx

**Comment by MichaelThessel** [comment]
Looks like this has been fixed by: #23862 This will still trigger scrolling but only if the extended content extends beyond the bottom of the viewport.
